File: SofiForce.Models/Models/EntityModels/ItemGroupModel.cs
using System;

namespace Models
{
    public class ItemGroupModel
    {
        public int ItemGroupId { get; set; }
        public string ItemGroupCode { get; set; }
        public string ItemGroupNameEn { get; set; }
        public string ItemGroupNameAr { get; set; }
        public string Icon { get; set; }
        public string Color { get; set; }
        public int? DisplayOrder { get; set; }
        public bool? CanEdit { get; set; }
        public bool? CanDelete { get; set; }
        public bool? IsActive { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/ItemGroupModel.ts
import { ItemModel } from "./itemModel";

export interface ItemGroupModel {
    itemGroupId: string;
    itemGroupCode: string;
    itemGroupNameEn: string;
    itemGroupNameAr: string;
    icon: string;
    color: string;
    displayOrder: number | null;
    canEdit: boolean | null;
    canDelete: boolean | null;
    isActive: boolean | null;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;

}